Rakefile

```diff
@@ -22,3 +24,8 @@ load 'rails/tasks/statistics.rake'
 Bundler::GemHelper.install_tasks
 
 Dir['lib/tasks/*.rake'].each { |r| load r }
+
+spec = Gem::Specification.find_by_name 'bower-rails'
+load "#{spec.gem_dir}/lib/tasks/bower.rake"
```

Comment:
Because bundler already puts the equivalent of 
`Gem::Specification.find_by_name('bower-rails')+'/lib'` into the load path. (More accurately: https://github.com/rharriso/bower-rails/blob/master/bower-rails.gemspec#L17)

Why do you think `require 'bower-rails'` works? Because there is a file `bower-rails.rb` in `/usr/lib/ruby/gems/bower-rails-1.2.3/lib` (or similar), and this path is in the global ruby load path.